css/alert.css

```diff
@@ -0,0 +1,38 @@
+.alertd {
+    padding: 1rem;
+    background-color: rgb(248, 66, 66);
+    color: white;
+    margin:1rem;
+  }
+  .alerts {
+    padding: 1rem;
+    background-color: rgb(116, 243, 116);
```

Comment:
Put all the colors in CSS variables which is used repeatedly.

Reply:
Okay, Will work on it.